### PR TITLE
WT-2459 - Force the CC tag under powerPC

### PR DESCRIPTION
**Pull request by daveh86**

### CODE REVIEW

File: build_posix/configure.ac.in [hunk 1]
```diff
@@ -34,6 +34,15 @@ AC_PROG_CC(cc gcc)
 AC_PROG_CXX(c++ g++)
 AM_PROG_AS(as gas)
 
+# Set the libtool tag flag on PowerPC
+AS_CASE([$host_cpu],
```
Comment (agorrod) [inline build_posix/configure.ac.in]: @daveh86 Is this only necessary when using clang? If so does it make sense to limit setting the flag to compiles using clang?

**Comment by daveh86** [comment]
@agorrod, changed as you suggested.

I also found that the issue with libtool can be worked around by providing any tag, rather than a valid one. 

For now, i've set the tag to nil and it works on PPC without changing the actual tag for the C compile tasks.

**Comment by agorrod** [comment]
@daveh86 Are you happy with the state of this now? i.e: is it ready for a final round of review?

**Comment by agorrod** [comment]
Thanks @daveh86 lgtm.

**Pull request merged**